test/parallel/test-fs-stat.js

```diff
@@ -72,14 +72,10 @@ fs.open('.', 'r', undefined, common.mustCall(function(err, fd) {
 // fstatSync
 fs.open('.', 'r', undefined, common.mustCall(function(err, fd) {
   let stats;
-  try {
-    stats = fs.fstatSync(fd);
```

Comment:
This line should be kept. Only the `try / catch` is not required. In case of an failure, it will throw the same error, no matter if the `try / catch` is there or not.